### DISCUSSION TITLE
Guild war option

### DISCUSSION THREAD

**Discussion by avallet1** [DiscordDungeons/Bugs#989] [Ideas]
The name talks by himself.
I don't really see why we can be ally with another guild if we cant be in a war with another.
Of course, it would complex to create but i think it would be a good option.

**Comment by Feathers8** [comment]
There has been a discussion about this a long time ago
They got stuck when it came to the reward

**Reply by BoomExplode** [reply]
i dont think it would be balanced tho
since theres the really big guilds or small guilds
people logically would just choose big guilds since it helps more than small guilds
so its would just be a one sided slaughter cause theres not really a factor of skill in drpg
except if there would be a new way to make it happen i dont think this is logical
